bip-0340.mediawiki

```diff
@@ -9,6 +9,7 @@
   Status: Final
   Type: Standards Track
   License: BSD-2-Clause
+  License-Code: BSD-2-Clause OR MIT OR CC0-1.0
```

Comment:
Nit, per BIP3 (proposed, so this could be done later):

`If there are multiple acceptable licenses, each should be on a separate line...the license list is an "OR choice", not an "AND also" requirement.`


```
License-Code: BSD-2-Clause
              MIT
              CC0-1.0
```

Reply:
The rules for licenses are the same per BIP2 or BIP3: if multiple licenses are provided fulfilling the terms of any license is sufficient, and the formatting is required to be as @jonatack suggests.

Reply:
Ok, but then I suggest that the BIP3 draft is changed to use [SPDX License Expressions](https://spdx.dev/learn/handling-license-info/) instead. These can involve OR and this is much clearer than just listing them: It's a widely used standard, and anyone reading it will understand it without the need to read BIP3. It should be easy enough to change the existing BIPs.